cas-client-core/pom.xml

```diff
@@ -10,8 +10,37 @@
     <packaging>jar</packaging>
     <name>Jasig CAS Client for Java - Core</name>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
     <dependencies>
         <dependency>
+            <groupId>commons-lang</groupId>
```

Comment:
Turns out the new configuration strategy components depend on commons-lang, so bumping to v3 is not feasible in the scope of this pull.